src/app-layout/utils/use-resize.tsx

```diff
@@ -72,8 +72,9 @@ function useResize(drawerRefObject: React.RefObject<HTMLDivElement>, drawerResiz
   }, [drawerSize, drawersMaxWidth, MIN_WIDTH]);
 
   const drawerResize = (resizeDetail: { size: number; id: string }) => {
-    const drawerItem = drawers?.items.find((item: DrawerItemProps) => item.id === resizeDetail.id);
+    const drawerItem = drawers?.items.find(item => item.id === resizeDetail.id);
     fireNonCancelableEvent(drawerItem?.onResize, resizeDetail);
+    fireNonCancelableEvent(drawers?.onResize, resizeDetail);
```

Comment:
🐛 Found the actual bug! There are two `onResize` callbacks, one on the drawer, the other one on the `drawers` object directly.

It was working in Classic, but not in Refresh. Fixed